spec/helpers/react_helper_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe ReactHelper do
+  describe '#search_results_layout' do
+    context 'when an affiliate has connections' do
+      let(:affiliate) { affiliates(:usagov_affiliate) }
+      let(:search)    { WebSearch.new(query: 'chocolate', affiliate: affiliate) }
+      let(:related_sites) do
+        [{
+          label: 'power',
+          link: 'http://test.host/search?affiliate=noaa.gov&query=chocolate'
+        }]
+      end
+
+      it 'sends related sites label and link to SearchResultsLayout component' do
```

Comment:
Can you add a test case where there are no related sites?

Reply:
I added it in here below